src/main/scala/io/druid/indexer/spark/SparkBatchIndexTask.scala

```diff
@@ -407,6 +414,46 @@ object SparkBatchIndexTask
         }
       )
 
+      log.info("Initializing emitter for metrics")
+
+      val service = task.getProperties.getProperty("com.metamx.emitter.service", "druid/batch/index")
+      val host = task.getProperties.getProperty("com.metamx.emitter.host", "127.0.0.1")
+
+      val emitter = new ServiceEmitter(service, host, createEmitter(task.getProperties, lifecycle))
```

Comment:
Yeah, I thought `EmittingLogger` uses the injected `ServiceEmitter`. If initializing the internal emitter is fine, I will use the injected one.